vms/rpcchainvm/vm_test.go

```diff
@@ -112,7 +112,7 @@ func TestHelperProcess(t *testing.T) {
 // interface are implemented.
 func TestVMServerInterface(t *testing.T) {
 	var wantMethods, gotMethods []string
-	pb := reflect.TypeOf((*vmpb.VMServer)(nil)).Elem()
+	pb := reflect.TypeFor[vm.VMServer]()
```

Comment:
It seems that we should also be using this below here and in the `codec/reflectcodec` package.